src/main/scala/ch/epfl/bluebrain/nexus/iam/io/EventSerializer.scala

```diff
@@ -16,7 +16,7 @@ import io.circe.generic.extras.Configuration
 import io.circe.generic.extras.semiauto._
 import io.circe.parser._
 import io.circe.syntax._
-import io.circe.{Decoder, Encoder}
+import io.circe.{Decoder, Encoder, Printer}
```

Comment:
I will look into it...

Reply:
I can't use it because in order to derive the encoders and decoders I need a reference to the actor system to pull the http uris from config, before the CoproductSerializer is implicitly generated.